### ISSUE TITLE
[Accessibility] Add roles and missing attributes on the markup

### ISSUE THREAD

**Issue by magsout** [webcompat/webcompat.com#334]
I think in the first time we can add the roles (banner, aside, etc ..)  and basic attributes of the elements, we have forgotten (like type = "button")(such as type = "button")
